static/js/containers/UserPage_test.js

```diff
@@ -149,7 +149,7 @@ describe("UserPage", () => {
 
       it('should let you add an education entry', done => {
```

Comment:
This is outside the scope of the issue but would you mind adding a test to save the dialog box? It looks like we don't have any coverage for the code which saves or cancels this dialog or the work history dialog